swift/Tests/CosmosSignerTests.swift

```diff
@@ -17,29 +17,60 @@ class CosmosSignerTests: XCTestCase {
         var sendCoinsMessage = TW_Cosmos_Proto_SendCoinsMessage()
         sendCoinsMessage.fromAddress = fromAddress
         sendCoinsMessage.toAddress = "cosmos1zt50azupanqlfam5afhv3hexwyutnukeh4c573"
-        var coin = TW_Cosmos_Proto_Coin()
-        coin.amount = 1
-        coin.denom = "muon"
-        sendCoinsMessage.amount = [coin]
+        var txAmount = TW_Cosmos_Proto_Amount()
+        txAmount.amount = 1
+        txAmount.denom = "muon"
+        sendCoinsMessage.amounts = [txAmount]
 
-        var feeCoin = TW_Cosmos_Proto_Coin()
-        feeCoin.amount = 1
-        feeCoin.denom = "muon"
+        var feeAmount = TW_Cosmos_Proto_Amount()
+        feeAmount.amount = 200
+        feeAmount.denom = "muon"
 
         var fee = TW_Cosmos_Proto_Fee()
         fee.gas = 200000
-        fee.amount = [feeCoin]
+        fee.amounts = [feeAmount]
 
         var signingInput = TW_Cosmos_Proto_SigningInput()
```

Comment:
same here, and we prefer:
```swift
CosmosSigningInput.with {
    $0.chainID = "gaia-13003"
    ....
}
```

Reply:
you can apply this pattern for other protobuf models, like `TW_Cosmos_Proto_StakeMessage`

Reply:
done in 3a15f10707237aa97320142331a0a3171c3e8cdb